### PR TITLE
fix: Text in sparkline does not render correctly in RTL mode

### PR DESCRIPTION
**Pull request by harsh9975**
## Previous Behavior
The 19.64 text for sparkline is misaligned and cropped in RTL mode.

## New Behavior
text alignment fixed.
Screenshot 2023-11-11 at 2 22 08 AM

## Related Issue(s)
#29804

### CODE REVIEW

File: packages/react-examples/src/react-charting/SparklineChart/SparklineChart.Basic.Example.tsx [hunk 1]
```diff
@@ -341,49 +341,49 @@ export class SparklineChartBasicExample extends React.Component<{}, ISparklineSt
         <table>
           <tbody>
             <tr>
-              <td style={{ paddingRight: '15px', paddingBottom: '5px', paddingTop: '5px' }}>Row 1</td>
+              <td style={{ padding: '5px 15px' }}>Row 1</td>
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/SparklineChart/SparklineChart.Basic.Example.tsx:344]: left padding is not needed.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a6ea546eff0df9f4f9cae403d4905ad81ff4e014'>a6ea546eff0df9f4f9cae403d4905ad81ff4e014</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-51f43c57-5f52-424a-a747-874776dcb240.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-51f43c57-5f52-424a-a747-874776dcb240.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29817&commitId=75a34446b451eab719011f2ed7bb74782a2a3ce2&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-SparkLineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|436| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/Sparkline/Sparkline.base.tsx [hunk 3]
```diff
@@ -125,7 +126,7 @@ export class SparklineBase extends React.Component<ISparklineProps, ISparklineSt
           )}
           {this.props.showLegend && this.props.data!.lineChartData![0].legend ? (
             <svg width={this.state._valueTextWidth} height={this.state._height} data-is-focusable={true}>
-              <text x="0%" dx={8} y="100%" dy={-5} className={classNames.valueText}>
+              <text x="0%" dx={this._isRTL ? 28 : 8} y="100%" dy={-5} className={classNames.valueText}>
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/Sparkline/Sparkline.base.tsx:129]: use text-anchor="end" property instead of dx.

**Review by AtishayMsft — APPROVED** [review]

File: change/@fluentui-react-charting-51f43c57-5f52-424a-a747-874776dcb240.json (hunk 2 above)
Comment (AtishayMsft) [reply change/@fluentui-react-charting-51f43c57-5f52-424a-a747-874776dcb240.json:1]: The PR fixes the text overflowing issue as shown in the original screenshot.

**Pull request merged**